auslib/admin/views/scheduled_changes.py

```diff
@@ -118,6 +125,44 @@ def _post(self, sc_id, transaction, changed_by):
         return Response(status=200)
 
 
+class SignoffsView(AdminView):
+    """/scheduled_change/:namespace/:sc_id/signoffs"""
+
+    def __init__(self, namespace, table):
+        self.namespace = namespace
+        self.path = "/scheduled_changes/%s/:sc_id/signoffs" % namespace
+        self.signoffs_table = table.scheduled_changes.signoffs
+        super(SignoffsView, self).__init__()
+
+    def _post(self, sc_id, transaction, changed_by):
```

Comment:
This gets enforced in the subclasses, actually. I think there was a complicated technical reason I couldn't put it here when I originally wrote this code...